developer_docs/development.md

```diff
@@ -87,7 +87,7 @@ Structure your commit message like this:
  
  - For common and reusable styles, write OOSCSS (Object-Oriented SCSS) with placeholders and mixins. For organizing styles, follow the [7-1 Pattern](https://sass-guidelin.es/#the-7-1-pattern) for Sass.
 
- - We're using [ES6](http://es6-features.org/) and transpiling to ES5 using [Babel](https://babeljs.io/). 
+ - We're using [ES6](http://es6-features.org/) and transpile to ES5 using [Babel](https://babeljs.io/). 
```

Comment:
I think 'transpiling' makes sense here, but could also see this sentence as a whole being rephrased as well!